src/spaceone/inventory/manager/workload/job_manager.py

```diff
@@ -0,0 +1,178 @@
+            try:
+                ##################################
+                # 1. Set Basic Information
+                ##################################
+                job_name = job.metadata.name
+                cluster_name = self.get_cluster_name(secret_data)
+                region = "global"
+
+                ##################################
+                # 2. Make Base Data
+                ##################################
+                # key:value type data need to be processed separately
+                # Convert object to dict
+                raw_data = job.to_dict()
+                raw_readonly = job.to_dict()
+                raw_data["metadata"]["annotations"] = self.convert_labels_format(
+                    raw_data.get("metadata", {}).get("annotations", {})
+                )
+                raw_data["metadata"]["labels"] = self.convert_labels_format(
+                    raw_data.get("metadata", {}).get("labels", {})
+                )
+                raw_data["spec"]["node_selector"] = self.convert_labels_format(
+                    raw_data.get("spec", {}).get("node_selector", {})
+                )
+                raw_data["spec"]["selector"][
+                    "match_labels"
+                ] = self.convert_labels_format(
+                    raw_data.get("spec", {}).get("selector", {}).get("match_labels", {})
+                )
+
+                raw_data["spec"]["template"]["metadata"][
+                    "annotations"
+                ] = self.convert_labels_format(
+                    raw_readonly.get("spec", {})
+                    .get("template", {})
+                    .get("metadata", {})
+                    .get("annotations", {})
+                )
+                raw_data["spec"]["template"]["metadata"][
+                    "labels"
+                ] = self.convert_labels_format(
+                    raw_readonly.get("spec", {})
+                    .get("template", {})
+                    .get("metadata", {})
+                    .get("labels", {})
+                )
+                raw_data["spec"]["template"]["spec"][
+                    "node_selector"
+                ] = self.convert_labels_format(
+                    raw_readonly.get("spec", {})
+                    .get("template", {})
+                    .get("spec", {})
+                    .get("node_selector", {})
+                )
+                raw_data["uid"] = raw_readonly.get("metadata", {}).get("uid", "")
+                raw_data["completions"] = self._get_completion(
+                    raw_readonly.get("status", {})
+                )
+                raw_data["age"] = self.get_age(
+                    raw_readonly.get("metadata", {}).get("creation_timestamp", "")
+                )
+                raw_data["pods"] = self._get_pods(
+                    list_all_pod,
+                    raw_readonly.get("metadata", {}).get("uid", ""),
+                )
```

Comment:
Overall, it shows a pattern in which a dictionary is expanded through an assignment expression.

My guess is how about a pattern that ties the below pattern together and updates an existing dictionary with a new one. It is expected that readability will increase due to indentation and also increase readability by grouping for the same scope.

**[AS-IS]**
```python
raw_data["spec"]["selector"][
                    "match_labels"
                ] = self.convert_labels_format(
                    raw_data.get("spec", {}).get("selector", {}).get("match_labels", {})
                )
```

**[TO-BE]**
```python
job_added = {}
if match_labels := self.convert_labels_format(raw_data.get("spec", {}).get("selector", {}).get("match_labels", {})):
    job_added["spec"]["selector"]["match_labels"] = match_labels

if ...

if ...

if ...

# finally
raw_data.update(job_added)
job_data = Job(raw_data, strict=False)
```

It might be worth considering once.